### DISCUSSION TITLE
Licencja

### DISCUSSION THREAD

**Discussion by czalaviek** [RybarskiDominik/GML-2021#33] [Ideas]
Zachęcam określić się na jakiej licencji kod i apka są rozpowszechniane.

**Comment by RybarskiDominik** [comment]
Na razie nie widzę takiej potrzeby, nie wybierając licencji mam pełne prawa autorskie do kodu.
Jeżeli ktoś chce rozwijać kod do prywatnych celów to nie mam nic przeciwko.

Wybrałbym jakąś otwartą licencję jakbym widział w tym sens.

**Reply by czalaviek** [reply]
Teraz  rozumiem intencję. Dzięki.
